integreat_cms/cms/models/users/organization.py

```diff
@@ -43,6 +42,8 @@ class Organization(AbstractBaseModel):
         default=timezone.now, verbose_name=_("creation date")
     )
 
+    website = models.URLField(max_length=250, null=True, verbose_name=_("website"))
```

Comment:
```suggestion
    website = models.URLField(max_length=250, verbose_name=_("website"))
```
`null=True` on a char field doesn't make much sense anyway. Empty values are converted from `None` to an empty string the first time the form is submitted...